tests/runtest.sh

```diff
@@ -384,6 +384,14 @@ function create_core_overlay {
 
     cp -f -v "$coreClrBinDir/"* "$coreOverlayDir/" 2>/dev/null
     cp -f -v "$mscorlibDir/mscorlib.dll" "$coreOverlayDir/"
+
+    if [ -e "$coreOverlayDir/coreclr_native_bin" ]; then
```

Comment:
Please make `$coreOverlayDir/coreclr_native_bin` a variable since it's referred to in so many places. I'd also consider renaming the directory itself since it's not just native bins.

Finally, can you add a comment (next to where you create the variable) explaining what this directory is for?